cmd/gopogh/main.go

```diff
@@ -89,3 +98,10 @@ func main() {
 		fmt.Println(string(j))
 	}
 }
+
+// dbVarProvided checks whether any of the database flags/environment variables are set
+func dbVarProvided(dbPath string, dbBackend string) bool {
+	return (dbPath != "" || os.Getenv("DB_PATH") != "") ||
```

Comment:
what happens if they provide partially ? like they only provide dbPath and not db_backend ?

Reply:
The .sql function calls fromenv which handles this case by exiting with an error telling them which variable is missing